compass/landice/suites/fct_integration.txt

```diff
@@ -0,0 +1,9 @@
+landice/dome/2000m/fo_fct_decomposition_test
+landice/dome/2000m/fo_fct_restart_test
+landice/dome/variable_resolution/fo_fct_decomposition_test
+landice/dome/variable_resolution/fo_fct_restart_test
+landice/greenland/fo_fct_decomposition_test
+landice/greenland/fo_fct_restart_test
+landice/thwaites/fct_decomposition_test
```

Comment:
I'm wondering if we ought to revamp the naming conventions to make it clear when 'fo' or other abbreviations refer to velocity solve vs advection vs other possible settings.  I imagine `compass list` is getting pretty busy.

Reply:
Agreed.

Reply:
Also, making a note that we'll need to change the names of the tests in the full_integration suite (and docs) accordingly (discovered this trying to run full_integration from this branch).